src/cache.ts

```diff
@@ -51,4 +51,13 @@ export class Cache implements Types.ICache {
   public has(key: string): boolean {
     return key in this._data;
   }
+
+  public purge(): void {
+    this._stats = {
+      hits: 0,
+      misses: 0,
+    };
```

Comment:
I suggest we do
```suggestion
    Object.assign(this._stats, {
      hits: 0,
      misses: 0,
    });
```

Could add a test for it.

or
```ts
this._stats.hits = 0;
this._stats.misses = 0;
```

Why? There is a public `stats` getter exposing the object, so we are leaking it, meaning the consumer might potentially operate on stale data, for example:
```ts
const { stats } = this.cache;

stats.hits; // prints 1 (for example);

this.cache.purge();

state.hits // still prints 1 instead of 0
```

Reply:
@P0lip Good catch! Fixed and updated the test to rely on the referenced getter.